documentation/API_documentation/NumberVerification_device_phone_number_User_Story.md

```diff
@@ -0,0 +1,10 @@
+# Number Verification Device Phone Number API User Story
+
+| **Item** | **Details** |
+| ---- | ------- |
+| ***Summary*** | As an enterprise application developer, I want to retrieve the phone number associated with the device from which the call was made, so that I can ensure that I obtain the correct phone number and avoid fraud e.g. identity theft. |
```

Comment:
I do not understand the use case. Stumbling about the word "call" here. 
Is the use  case this?: 

`A user calls the enterprise hotline, and as an enterprise application developer, I want to retrieve the phone number associated with the device from which the call was made, so that I can ensure that I obtain the correct phone number and avoid fraud e.g. identity theft.`

If this is the use case, then how is the access token obtained? 

The scope of NV is
> verify the phone number associated with the SIM used in the device connected to the mobile **data** network.

From that reasoning above I think the work "call" is wrong, right?

Maybe the use case(s) should start with how the access token obtained?

Reply:
Hi @gregory1g, let me ping you for this topic since from TEF side @jgarciahospital is OOO right now. Thanks

Reply:
Call refers to the API call, not any other phone call. If there is any other more appropriate term to be used we can update it.

Reply:
What about **api call** instead of **call** ?

Reply:
@bigludo7 @jgarciahospital @fernandopradocabrillo 
please see my suggestion https://github.com/camaraproject/NumberVerification/pull/129#discussion_r1727232532

Reply:
Has this been updated "API call" instead of "call", it still seems to be "call"?

Reply:
@AxelNennker proposed an alternative text yesterday and I have applied it